docs/rules/security/avoid-low-level-calls.md

```diff
@@ -28,12 +28,21 @@ This rule accepts a string option of rule severity. Must be one of "error", "war
 
 
 ## Examples
+### 👍 Examples of **correct** code for this rule
+
+#### Using low level calls to transfer funds
+
+```solidity
+anyAddress.call{value: 1 ether}("");
+anyAddress.call.value(code)();
```

Comment:
`a.call.value` is deprecated.

```suggestion
anyAddress.call{value: 1 ether}("");
```

Reply:
Notice that this means updating the implementation and the tests.

Reply:
corrected